src/resources/birthdayTimer.ts

```diff
@@ -0,0 +1,70 @@
+import { bdayDates, bdayTimes, bdayChannels } from '../slashcommands/Birthday';
+import cron from 'node-cron';
+import { Bot } from '../Bot';
+import { MessageEmbed, TextChannel, User } from 'discord.js';
+import Enmap from 'enmap';
+
+export let bdayCrons = new Enmap();
+
+export async function birthdayTimer(guild: string, bot: Bot): Promise<void> {
+	//grab the stored infostring from the enmap, if empty: return
+	let infoString = bdayTimes.ensure(guild, '') as String;
+	if (infoString == '') {
+		bot.logger.warn(`Embed value did not exist for guild: ${guild}.`); 
+		return;
+	}
+	//split the infoarray to be interpreted and interpolate the cron string to be accurate
+	let infoArray = infoString.split('-'); //[0]=MM [1]=HH [2]=DATEMOD [3]=Timezone for notes
+	let cronTime = `${infoArray[0]} ${infoArray[1]} * * *`;
+	console.log(infoArray);
+
+	//if a cronJob already exists for a guild we will want to end it before scheduling a new one.
+	let task = bdayCrons.get(guild);
+	if (task) {
+		bot.logger.warn('unscheduling cron!');
```

Comment:
This definitely doesn't need to be a warn if it's performing an action we expect to happen sometimes